gobblin-docs/user-guide/Docker-Integration.md

```diff
@@ -94,9 +98,7 @@ Similar to standalone working directory settings:
 
 Run these commands to start the docker image:
 
-`docker pull gobblin/gobblin-service:alpine-gaas-latest`
-
-`docker run -p 6956:6956 -v GAAS_JOB_DIR:/tmp/gobblin-as-service/jobs -v LOCAL_DATAPACK_DIR:/tmp/templateCatalog gobblin/gobblin-service:alpine-gaas-latest`
+`docker run -p 6956:6956 -v GAAS_JOB_DIR:/tmp/gobblin-as-service/jobs -v LOCAL_DATAPACK_DIR:/tmp/templateCatalog apache/gobblin --mode gobblin-as-service`
```

Comment:
Need to add $ sign for the directories exports like this:
docker run -p 6956:6956 -v $GAAS_JOB_DIR:/tmp/gobblin-as-service/jobs -v $LOCAL_DATAPACK_DIR:/tmp/templateCatalog apache/gobblin --mode gobblin-as-service